desktop/docs/secrets.md

```diff
@@ -14,10 +14,11 @@ and click on the create button. The other fields may be left blank.
 In the end, your application details should look similar to:
 ![application details](https://cldup.com/Juw-5uEHlR.png)
 
-After your application is created, copy your client id and client secret, and add a `calypso/config/secrets.json` file:
+After your application is created, copy your client id and client secret, and either add or update `/config/secrets.json` file and be sure to include the following properties:
```

Comment:
I ran into this, it’s a little bit weird that I have to create a new file (there isn’t a default structure) and that the name of the file is as generic as `secrets.json`.